Custom_GruppeAdler/streetsTanoa.sqf

```diff
@@ -0,0 +1,14015 @@
+1677575,
+1680973,
+1677622,
+1677623,
+1676276,
+1676277,
+1688359,
+1688360,
+1675796,
+1675105,
+1675797,
+1680662,
+1680663,
+1680614,
+1680615,
+1680699,
+1680698,
+1471096,
+1680697,
+1680712,
+1680710,
+1680711,
+1471085,
+1471084,
+1680746,
+1681555,
+1681502,
+1680936,
+1678499,
+1678500,
+1677574,
+1677573,
+1680971,
+1680972,
+1677624,
+1676275,
+1676274,
+1676273,
+1680910,
+1688358,
+1675795,
+1675794,
+1675106,
+1680616,
+1680617,
+1680619,
+1680618,
+1680620,
+1680621,
+1680622,
+1680623,
+1680701,
+1680700,
+1680743,
+1680744,
+1680745,
+1681556,
+1681501,
+1680937,
+1678501,
+1677572,
+1680970,
+1676272,
+1676271,
+1675812,
+1680911,
+1680912,
+1688356,
+1688357,
+1675793,
+1675107,
+1680624,
+1680625,
+1680702,
+1680703,
+1680742,
+1681557,
+1681500,
+1681499,
+1680939,
+1680938,
+1678502,
+1677571,
+1680969,
+1675814,
+1675813,
+1676269,
+1676270,
+1680913,
+1688354,
+1688355,
+1675791,
+1675792,
+1675108,
+1675109,
+1680626,
+1680627,
+1680704,
+1471087,
+1680741,
+1680739,
+1680740,
+1681558,
+1681498,
+1680940,
+1678503,
+1680956,
+1680957,
+1680958,
+1677570,
+1677569,
+1680959,
+1677436,
+1680960,
+1680961,
+1680968,
+1675815,
+1676268,
+1676267,
+1692624,
+1680914,
+1680915,
+1688352,
+1688353,
+1675789,
+1675732,
+1675790,
+1675110,
+1680628,
+1680629,
+1471086,
+1680736,
+1680738,
+1680737,
+1681559,
+1681560,
+1681496,
+1681497,
+1680941,
+1678504,
+1680955,
+1680954,
+1677437,
+1677438,
+1680962,
+1680963,
+1680964,
+1677444,
+1680967,
+1675816,
+1675817,
+1676266,
+1676265,
+1680916,
+1688350,
+1688351,
+1675788,
+1675787,
+1675733,
+1675734,
+1675111,
+1680630,
+1680735,
+1681561,
+1681495,
+1680942,
+1680943,
+1678505,
+1680952,
+1680953,
+1677439,
+1677440,
+1677517,
+1677516,
+1677441,
+1677442,
+1677443,
+1680965,
+1680966,
+1680105,
+1680104,
+1675818,
+1676263,
+1676264,
+1680917,
+1680918,
+1688347,
+1688348,
+1688349,
+1675786,
+1675735,
+1675113,
+1675112,
+1691463,
+1680631,
+1680734,
+1680733,
+1678191,
+1678190,
+1681562,
+1678189,
+1681494,
+1680944,
+1680945,
+1678506,
+1678507,
+1680946,
+1680948,
+1680947,
+1680949,
+1680950,
+1680951,
+1677482,
+1677481,
+1677518,
+1680106,
+1676420,
+1675819,
+1675820,
+1676419,
+1676261,
+1676262,
+1676260,
+1676450,
+1676451,
+1688345,
+1688346,
+1680919,
+1675783,
+1675785,
+1675784,
+1675729,
+1675736,
+1675730,
+1675731,
+1675114,
+1680633,
+1680632,
+1680732,
+1691326,
+1678193,
+1678192,
+1681493,
+1677485,
+1677484,
+1677483,
+1677520,
+1677519,
+1680107,
+1676423,
+1676421,
+1676422,
+1675822,
+1675821,
+1676259,
+1676258,
+1676257,
+1676449,
+1688343,
+1688344,
+1675728,
+1675738,
+1675737,
+1675115,
+1680634,
+1680731,
+1678195,
+1678194,
+1681563,
+1681492,
+1677488,
+1677487,
+1677486,
+1677522,
+1677521,
+1680108,
+1676424,
+1675823,
+1676256,
+1676255,
+1676254,
+1676448,
+1676447,
+1688342,
+1675726,
+1675727,
+1675739,
+1675740,
+1675741,
+1675116,
+1680635,
+1680730,
+1678196,
+1681564,
+1681491,
+1677491,
+1677490,
+1677489,
+1677525,
+1677524
```

Comment:
tanoa streets are not necessary at all. are not used in any form and probably never